package-lock.json

```diff
@@ -15,6 +15,7 @@
         "@actions/exec": "^1.1.1",
         "@actions/glob": "^0.4.0",
         "@actions/io": "^1.1.3",
+        "smol-toml": "^1.1.1",
         "toml": "^3.0.0"
```

Comment:
I’m wondering why this is still here?

Reply:
Seems that `npm` doesn't remove this automatically.